pkg/redshiftsql/ddl.go

```diff
@@ -0,0 +1,199 @@
+package redshiftsql
+
+import (
+	"database/sql"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/pingcap-inc/tidb2dw/pkg/tidbsql"
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	timodel "github.com/pingcap/tidb/parser/model"
+	"github.com/pingcap/tiflow/pkg/sink/cloudstorage"
+	"go.uber.org/zap"
+)
+
+func GetColumnModifyString(diff *tidbsql.ColumnDiff) (string, error) {
+	strs := make([]string, 0, 3)
+	if diff.Before.Tp != diff.After.Tp || diff.Before.Precision != diff.After.Precision || diff.Before.Scale != diff.After.Scale {
+		colStr, err := GetRedshiftTypeString(*diff.After)
+		if err != nil {
+			return "", errors.Trace(err)
+		}
+		strs = append(strs, fmt.Sprintf("COLUMN %s", colStr))
+	}
```

Comment:
report error when modify column type.

Reply:
It will report an error in line 84. I think we can keep this part in case the Redshift can support column modification in the future.

Reply:
ok